ORStools/gui/ORStoolsDialogUI.ui

```diff
@@ -387,10 +387,10 @@
          <property name="checked">
           <bool>false</bool>
          </property>
-         <property name="collapsed">
```

Comment:
Do you have an explanation what these `stdset="0"` do?

Reply:
Oversaw that this changed... They appear to set the standard value for the property, with `0` meaning `False`. Should i remove them again? I think they appeared after saving in QtCreator.

Reply:
Ah. I guess so, since we don't want a value set?

Reply:
Done